.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+## What?
+...
+
+## Why?
+...
+
+## Testing / Proof
+...
+
+ping {suggested reviewers}
```

Comment:
In that case perhaps this should ping @ frontend by default?

Reply:
Yep, good idea. I'll update.